IssueTracker/IssueTracker.FileSystem/Data/IData/IMetaData.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Models.Request;
+using Models.Response;
+
+namespace IssueTracker.FileSystem.Data.IData;
```

Comment:
simplify the namespace: IssueTracker.FileSystem

Reply:
I simplified

Reply:
not everywhere